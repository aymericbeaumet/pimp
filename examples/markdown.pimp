#!/usr/bin/env pimp --run

$input := `
# foobar

- you have to do this
- and this
`

MarkdownRender $input
