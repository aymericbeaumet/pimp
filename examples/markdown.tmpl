#!/usr/bin/env pimp --render

{{- $input := `
# foobar

- you have to do this
- and this
` -}}

{{- MarkdownRender $input -}}
