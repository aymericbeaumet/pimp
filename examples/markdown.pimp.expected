<h1 id="foobar">foobar</h1>
<ul>
<li>you have to do this</li>
<li>and this</li>
</ul>
